DebugSwift/Sources/Base/FeatureBase.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FeatureBase.swift
+//  DebugSwift
+//
+//  Created by Matheus Gois on 11/06/24.
+//
+
+import UIKit
+
+public protocol MainFeatureType {
+    var controllerType: DebugSwiftMainFeature { get }
```

Comment:
Shouldn't we use the new type alias here
```suggestion
    var controllerType: DebugSwiftFeature { get }
```

Reply:
The old name was `DebugSwiftFeatures`:

<img width="969" alt="image" src="https://github.com/DebugSwift/DebugSwift/assets/31082311/85c6cd11-b9c4-4788-86e4-ac294ab5549d">


You propose a diferente name?

Reply:
I can't unify because they have different purposes:

MainFeatures (Entire screens)
Features (Isolated features)

Reply:
I believe we should have some sort of hierarchy:

MainFeatures -> FeatureSet - for example - .network
Features -> Functionaly - for example - .swizzleLocation

Also, based on the `typealias` declaration I thought that we should start using `DebugSwiftFeatures` in all places where `DebugSwiftMainFeature` was used (This is what I've proposed as minor changes.)